### PR TITLE
Pull Request - nav_branch

### PR DESCRIPTION
**Pull request by tawakuliKH**
Added html and css files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <link rel="stylesheet" href="style.css">
+    <title>Portofolio</title>
+</head>
+
+<body>
+
+    <header>
+        <div class="header">
+            <div class="mylogo"><a class="logo">My Logo</a></div>
+            <div class="vector"><img src="assets/vector.svg" alt="vector"></div>
```
Comment (Kossi-stack) [inline index.html:17]: - [x] Kindly use a nav tag for hamburger
The nav tag is for semantics reasons and the right position for convention and user experience for mobile devices.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    </header>
+
+    <section class="main">
+        <h1 class="intro">I'm Morteza Glad to See You!</h1>
+        <p class="paragraph"> I’m a software developer! I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+            to contact me.</p>
+        <div class="connect">LET'S CONNECT</div>
+        <div class="social">
```
Comment (Kossi-stack) [inline index.html:28]: - [ ] Kindly put The list of icons in a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)
the `a` tag will let you connect your social networks with the icons, using href.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portofolio</title>
+</head>
+
+<body>
+
+    <header>
+        <div class="header">
+            <div class="mylogo"><a class="logo">My Logo</a></div>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly remove the div and leave it in a tag
because of convention, semantics, and user experience(UX).

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @tawakuliKH,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly edit the PR's title to something more descriptive. it can be "Portfolio: setup PR"

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Ibizugbe — APPROVED** [review]
Hi @tawakuliKH 👋, 

BigTvGIF (2)


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅
- linters are passing👍
- commit messages are meaningful👍
- github flow has been used👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**